content/3-lifecycle/1-on-mount/emberPolaris/page-title.gjs

```diff
@@ -0,0 +1,9 @@
+import Component from "@glimmer/component";
+
+export default class PageTitle extends Component {
```

Comment:
class not needed here, see: https://tutorial.glimdown.com/2-reactivity/10-synchronizing-external-state